tests/integration/introduction/test_introduction.py

```diff
@@ -57,3 +57,19 @@ def test_start_survey_sets_started_at(self):
         started_at_datetime = datetime.strptime(actual['started_at'], '%Y-%m-%dT%H:%M:%S.%f')
 
         self.assertIsNotNone(started_at_datetime)
+
+    def test_legal_basis_should_be_visible(self):
+        # Given survey with legal_basis for completion
+        self.launchSurvey('1', '0112')
```

Comment:
We don't use live surveys in tests. Can you change this to one of our test schemas please. Same with mbs below.

Reply:
Yes please. Boy scout rule.